eng/pipelines/common/global-build-job.yml

```diff
@@ -54,7 +60,7 @@ jobs:
     - ${{ if eq(parameters.isOfficialBuild, true) }}:
       - template: /eng/pipelines/common/restore-internal-tools.yml
 
-    - ${{ if in(parameters.osGroup, 'OSX', 'iOS', 'tvOS') }}:
+    - ${{ if or(in(parameters.osGroup, 'OSX', 'iOS', 'tvOS'), and(eq(parameters.osGroup, 'Android'), eq(parameters.isOfficialBuild, false))) }}:
```

Comment:
@directhex @premun do we need only to run tests or also for the final shipping product?

Reply:
I have no idea about this, just copied it from @steveisok